types/jest-environment-puppeteer/index.d.ts

```diff
@@ -44,11 +50,47 @@ interface JestPuppeteer {
     debug(): Promise<void>;
 }
 
+export interface Timer {
+    id: number;
+    ref: () => Timer;
+    unref: () => Timer;
+}
+
+export interface Global extends GlobalType.Global {
+    browser: Browser;
+    context: Context;
+    page: Page;
+    jestPuppeteer: JestPuppeteer;
+}
+
+/** Note: TestEnvironment is sandboxed. Each test suite will trigger setup/teardown in their own TestEnvironment. */
+declare class PuppeteerEnvironment implements JestEnvironment {
+  context: Context | null;
+  fakeTimers: FakeTimers<Timer> | null;
+  global: Global;
+  moduleMocker: ModuleMocker | null;
+  constructor(config: Config.ProjectConfig);
+
+  /**
+   * Setup runs when the environment is being spun up, generally before each test suite
+   * You should always call `await super.setup()` in here
+   */
+  setup(): Promise<void>;
+
+  /**
+   * Teardowns runs as the environment is being torn down, generally after each test suite.
+   * You should always call `await super.tearDown()` in here
+   */
+  teardown(): Promise<void>;
+  runScript(script: Script): any;
+  handleTestEvent?(event: Circus.Event, state: Circus.State): void;
+}
+
 declare global {
     const browser: Browser;
     const context: BrowserContext;
     const page: Page;
     const jestPuppeteer: JestPuppeteer;
 }
 
-export {};
+export default PuppeteerEnvironment;
```

Comment:
It seems like the [underlying library](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/index.js#L1) is not using `export default`, so this .d.ts file should not either. [This](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-modifying-module-d-ts.html) is the template you should be using. 

```suggestion
export = PuppeteerEnvironment;
```